pkg/cache/v3/linear.go

```diff
@@ -28,15 +28,37 @@ import (
 
 // cachedResource is used to track resources added by the user in the cache.
 // It contains the resource itself and its associated version (currently in two different modes).
-// TODO(valerian-roche): store serialized resource if already marshaled to compute the version.
 type cachedResource struct {
 	types.Resource
 
 	// cacheVersion is the version of the cache at the time of last update, used in sotw.
 	cacheVersion string
-	// resourceVersion is the version of the resource itself (built through stable marshaling).
-	// It is only set if computeResourceVersion is set to true on the cache.
-	resourceVersion string
+	// stableVersion is the version of the resource itself (a hash of its content after deterministic marshaling).
```

Comment:
Could we clarify if stableVersions is only used for delta, e.g. cacheVersion for SOTW and stableVersion for delta? I believe this is the case but I'm not sure.

Reply:
Hmm I guess this is changed in other PRs, where stableVersion is also used for STOW

Reply:
I did not want to specify this here as it is imo irrelevant in the context of the cachedResource (it is relevant to the calls to `getVersion`)
Also as noted for now it's delta only, but no longer true when #12 gets through